### PR TITLE
Don't assign kaspth from Active Record PRs.

### PR DESCRIPTION
**Pull request by kaspth**
Young schmo doesn't feel comfortable ( ͡° ͜ʖ ͡°)'ing the record.

@rafaelfranca, did I do it right?

### CODE REVIEW

File: highfive/configs/rails.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
     "groups": {
         "all": ["core_triage", "commiters"]
     },
+    "dirs": {
+      "activerecord": ["activerecord"]
```
Comment (rafaelfranca) [inline highfive/configs/rails.json]: `["core_triage", "activerecord"]`

File: highfive/configs/_global.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
         "core": ["@carlosantoniodasilva", "@chancancode", "@dhh", "@fxn", "@guilleiguaran", "@jeremy", "@josevalim", "NZKoz", "@pixeltrix", "@rafaelfranca", "@senny", "@spastorino", "@tenderlove"],
         "core_triage": ["@chancancode", "@pixeltrix", "@rafaelfranca", "@senny"],
         "commiters": ["@arthurnn", "@eileencodes", "@kaspth", "@matthewd", "@schneems", "@sgrif"],
+        "activerecord": ["@arthurnn", "@eileencodes", "@matthewd", "@schneems", "@sgrif"]
```
Comment (rafaelfranca) [inline highfive/configs/_global.json]: remember to remove matthew from here.

**Comment by rafaelfranca** [comment]
Needs a rebase. After that :shipit:

**Comment by kaspth** [comment]
Noticed, thanks 👍

**Pull request merged**